roles/rsyslog/templates/global.j2

```diff
@@ -0,0 +1,38 @@
+{% set ns0 = namespace(ca_cert_path = __rsyslog_default_pki_path + __rsyslog_default_pki_cert_dir + __rsyslog_default_pki_ca_cert_name) %}
+{% set ns1 = namespace(cert_path = __rsyslog_default_pki_path + __rsyslog_default_pki_cert_dir + __rsyslog_default_pki_cert_name) %}
+{% set ns2 = namespace(key_path = __rsyslog_default_pki_path + __rsyslog_default_pki_key_dir + __rsyslog_default_pki_key_name) %}
```

Comment:
You can use a single `namespace` object for this - https://jinja.palletsprojects.com/en/2.11.x/templates/#namespace
```
{% set ns = namespace() %}
{% set ns.ca_cert_path = __rsyslog_default_pki_path + .... %}
{% set ns.cert_path = .... %}
{% set ns.key_path = %}